Formula/l/logcli.rb

```diff
@@ -1,3 +1,6 @@
+# frozen_string_literal: true
+
+# Run LogQL queries against a Loki server
```

Comment:
```suggestion
```

Reply:
It was suggested by brew style. That's the reason why I added those lines.

Reply:
Did you run `brew style ./path/to/logcli.rb` where `logcli.rb` is outside the tap? For formulae we use `brew style logcli`.